browser/ui/tabs/features.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_TABS_FEATURES_H_
+#define BRAVE_BROWSER_UI_TABS_FEATURES_H_
+
+#include "base/feature_list.h"
+
+namespace tabs::features {
```

Comment:
How about making one features file here by moving another feature(`kBraveChangeActiveTabOnScrollEvent`) here?
and then `browser/ui/views/tabs/features.cc` could be renamed such as `browser/ui/views/tabs/utils.cc`?

Reply:
In the previous review, I got feedback that we should keep a feature in views/ if the feature is used only by the views/. So I feel like keeping kBraveChangeActiveTabOnScrollEvent there for now.

Reply:
Well.. it's weird to me to have two feature files for one feature(`tabs`).
If we move it also, I think we can avoid view path including(brave/browser/ui/views/tabs/features.h) at `brave/browser/about_flags.cc` and can use `brave/browser/ui/tabs/features` instead.
I think it's more cleaner?

Reply:
Yeah, that's what I prefer, too. Let's get this done then.